RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -244,6 +248,7 @@ public ObservableCollection<CodeExplorerItemViewModel> Projects
                 _projects = new ObservableCollection<CodeExplorerItemViewModel>(value.OrderBy(o => o.NameWithSignature));
                 
                 OnPropertyChanged();
+                OnPropertyChanged("TreeViewVisibility");
```

Comment:
why?

Reply:
The tree view is layered above the "Refresh" message.  It doesn't become visible until the TreeView has a project to display.  This changes that visibility.

Reply:
that was a (somewhat hidden) prompt to add a comment.